scripts/hotfix.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+"""
+Script to generate alton and git commands for executing hotfixes
+Commands for:
+  - cutting amis
+  - creating hotfix tag
+
+The script should be run with the hotfix's git hash as a command-line argument.
+Otherwise, the script will prompt the user for the hotfix hash.
+i.e. `python scripts/hotfix.py <hotfix hash>`
+"""
+import datetime
+import sys
+
+if len(sys.argv) == 2:
+    hotfix_hash = sys.argv[1]
+else:
+    hotfix_hash = raw_input("Please enter hotfix's git hash: ")
+
+print ""
```

Comment:
Can we use the print function, rather than the print keyword? Just put `from __future__ import print_function` at the top of the file ([`__future__` import docs are here](https://docs.python.org/2.7/library/__future__.html)), and then use `print` like a function (with parentheses) instead of like a keyword.